src/hotspot/share/opto/parse2.cpp

```diff
@@ -1550,9 +1577,65 @@ void Parse::do_if(BoolTest::mask btest, Node* c) {
   } else {
     adjust_map_after_if(untaken_btest, c, untaken_prob, next_block);
   }
+
+  if (do_stress_trap) {
+    stress_trap(iff, counter, incr_store);
+  }
+}
+
+void Parse::stress_trap(IfNode* orig_iff, Node* counter, Node* incr_store) {
+  // Search for an unstable if trap
+  CallStaticJavaNode* trap = nullptr;
+  for (int i = 0; i <= 1; ++i) {
+    Node* out = orig_iff->raw_out(i)->find_out_with(Op_CallStaticJava);
+    if (out != nullptr && out->isa_CallStaticJava() && out->as_CallStaticJava()->is_uncommon_trap()) {
```

Comment:
You don't need `out->isa_CallStaticJava()` because `find_out_with()` will return `nullptr` in other cases.

Reply:
Good catch. Fixed.